src/core/Form/SearchInput/__snapshots__/SearchInput.test.tsx.snap

```diff
@@ -90,41 +90,49 @@ exports[`calling render with the same component on the same container does not r
   color: inherit;
   background: none;
   cursor: inherit;
-  display: inline;
+  display: block;
   max-width: 100%;
   word-wrap: normal;
   word-break: normal;
   white-space: normal;
 }
 
-.c3 {
+.c5 {
+  display: inline-block;
+  vertical-align: baseline;
+}
+
+.c0 .fi-search-input_input-container {
+  position: relative;
+}
+
+.c0 .fi-search-input_input {
+  padding-right: 40px;
+}
+
+.c0 .fi-search-input_icon {
   position: absolute;
-  -webkit-clip: rect(0 0 0 0);
-  clip: rect(0 0 0 0);
-  height: 1px;
-  width: 1px;
-  margin: -1px;
-  padding: 0;
-  border: 0;
-  overflow: hidden;
+  top: 50%;
+  right: 16px;
+  margin-top: -0.5em;
 }
```

Comment:
I actually don't really get where some of these changes came from. Enlighten me if you can. Things look normal.

Reply:
Changing style import / adding order might affect things, but might as well have nothing to do with these...